python/ray/_private/worker.py

```diff
@@ -2274,6 +2274,7 @@ def connect(
     entrypoint: str = "",
     worker_launch_time_ms: int = -1,
     worker_launched_time_ms: int = -1,
+    debug_source: str = "",
```

Comment:
shall we set this to `"python runtime"`?

Reply:
I prefer not to add any default meaningful value. The reason is:
- The field is not used at runtime, only for manual purpose, which means even if we forget to assign or fails to assign, we cannot detect it with CI, thus hard to maintain.
- For these type of fields, it's better and more important to keep them always correct (I thought about RAY_CHECK against emptiness, but don't think it adds much value).